skins/cat17/templates/Update_Address.html.twig

```diff
@@ -8,10 +8,10 @@
 {% endblock %}
 
 {% block scripts %}
-	<script src="{$ basepath $}{$ '/skins/cat17/scripts/address-change/js/app.js' | prefix_file $}"
+	<script src="{$ basepath $}{$ '/skins/cat17/scripts/address-change/app.js' | prefix_file $}"
```

Comment:
just noticed `js` folder is removed from the path here but why? Wouldn't that break without changing any `vue-cli` configs?